schemas/Clinical/index.js

```diff
@@ -293,7 +301,7 @@ const convertClinicalSubmissionEntityToGql = (type, entity) => {
       get(entity, 'dataErrors', []).map(error => convertClinicalSubmissionErrorToGql(error)),
     dataUpdates: () =>
       get(entity, 'dataUpdates', []).map(update => convertClinicalSubmissionUpdateToGql(update)),
-    createdAt: new Date(), // this is a place holder for now
+    createdAt: entity.createdAt || null,
```

Comment:
is `entity.createdAt` a `Date` instance?

Reply:
yes, it is a Date instance in [clinical](https://github.com/icgc-argo/argo-clinical/blob/86c8626ed4e95804eba485ed74db33b2238db13f/src/submission/submission-entities.ts#L156)

Reply:
I think we'd be losing that context through the JSON serialization though :/

Reply:
maybe `entity.createdAt ? new Date(entity.createdAt) : null` to be safe? :P